test/module/irohad/torii/command_sync_client_test.cpp

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "main/server_runner.hpp"
+#include "module/irohad/torii/torii_mocks.hpp"
+#include "torii/command_client.hpp"
+
+using testing::_;
+using testing::Invoke;
+using testing::Return;
+
+class SyncClient : public testing::Test {
+ public:
+  void SetUp() override {
+    runner = std::make_unique<ServerRunner>(ip + ":0");
+    server = std::make_shared<iroha::torii::MockCommandServiceTransport>();
+    runner->append(server).run().match(
+        [this](iroha::expected::Value<int> port) { this->port = port.value; },
+        [](iroha::expected::Error<std::string> err) { FAIL() << err.error; });
+    runner->waitForServersReady();
+  }
+
+  std::unique_ptr<ServerRunner> runner;
+  std::shared_ptr<iroha::torii::MockCommandServiceTransport> server;
+
+  const std::string ip = "127.0.0.1";
+  int port;
+};
+
+/**
+ * @given command client
+ * @when different methods of the object is called
+ * @then the desired behavior achieved
+ */
+TEST_F(SyncClient, CommandClient) {
+  iroha::protocol::TxStatusRequest tx_request;
+  tx_request.set_tx_hash(std::string(32, '1'));
+  iroha::protocol::ToriiResponse toriiResponse;
+
+  auto client1 = torii::CommandSyncClient(ip, port);
+  // Copy ctor
+  torii::CommandSyncClient client2(client1);
+  // copy assignment
+  auto client3 = client2;
+  // move ctor
+  torii::CommandSyncClient client4(std::move(client3));
+  // move assignment
```

Comment:
Do all these ctors and assignments have to be tested in a single test? And client2 is not tested after copy assignment for example.

Reply:
Decided to remove this functionality because it's useless